### PR TITLE
add tags from pool to machine class

### PR DESCRIPTION
**Pull request by shaoyongfeng**
**How to categorize this PR?**
/area control-plane
/kind enhancement
/platform alicloud

**What this PR does / why we need it**:
Labels from work pool is not added to ECS instance. In this PR, we add these labels to machineclass so that they will be added to ECS as tag
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other user
Copy labels from work pool to ECS instance Tags.
```

### CODE REVIEW

File: pkg/controller/worker/machines.go [hunk 1]
```diff
@@ -214,6 +217,7 @@ func computeDisks(namespace string, pool extensionsv1alpha1.WorkerPool) (map[str
 			if vol.Type != nil {
 				dataDisk["category"] = *vol.Type
 			}
+			dataDisk["encrypted"] = true
```
Comment (jia-jerry) [inline pkg/controller/worker/machines.go:220]: Please handle encrypted logic in other logic. There will be side effect when a shoot is reconciled.

File: pkg/controller/common/encryptimage.go [hunk 2]
```diff
@@ -41,11 +41,11 @@ func UseEncryptedSystemDisk(volume interface{}) (bool, error) {
 	}
 
 	if v, ok := volume.(*extensionsv1alpha1.Volume); ok {
-		return v.Encrypted != nil && *v.Encrypted, nil
+		return (v.Encrypted == nil || v.Encrypted != nil && *v.Encrypted), nil
```
Comment (jia-jerry) [inline pkg/controller/common/encryptimage.go:44]: Please handle encrypted logic in other logic. There will be side effect when a shoot is reconciled.

File: pkg/controller/common/encryptimage.go [hunk 3]
```diff
@@ -41,11 +41,11 @@ func UseEncryptedSystemDisk(volume interface{}) (bool, error) {
...
 	}
 
 	if v, ok := volume.(*gcorev1beta1.Volume); ok {
-		return v.Encrypted != nil && *v.Encrypted, nil
+		return (v.Encrypted == nil || v.Encrypted != nil && *v.Encrypted), nil
```
Comment (jia-jerry) [inline pkg/controller/common/encryptimage.go:48]: Please handle encrypted logic in other logic. There will be side effect when a shoot is reconciled.

**Review by jia-jerry — CHANGES_REQUESTED** [review]

**Review by jia-jerry — APPROVED** [review]

**Review by EmoinLanyu — APPROVED** [review]
/lgtm

**Pull request merged**